assets/views/home/header.ejs

```diff
@@ -55,7 +55,7 @@
 
       <ul class="nav navbar-nav hidden" ng-hide="user" ng-class="loaded ? '' : 'hidden'">
         <li><a href="/info">Information</a></li>
-        <li><a href="/auth/reddit/">Login with Reddit</a></li>
+        <li><a href="/auth/reddit{{querystring}}">Login with Reddit</a></li>
```

Comment:
This won't change while we are on the page, will it? It's based on the query string from the url, so can you render it serverside instead? I've just changed the header to work mostly without javascript (if you ignore the dropdowns).